### PR TITLE
Simpler cookie policy

### PR DESCRIPTION
**Pull request by jonathanKingston**
## Description:

https://app.asana.com/0/0/1202348774384668/f

## Automated tests:
- [ ] Unit tests
- [ ] Integration tests

###### Reviewer Checklist:
- [ ] **Ensure the PR solves the problem**
- [ ] **Review every line of code**
- [ ] **Ensure the PR does no harm by testing the changes thoroughly**
- [ ] **Get help if you're uncomfortable with any of the above!**
- [ ] Determine if there are any quick wins that improve the implementation

###### PR Author Checklist:
- [ ] Get advice or leverage existing code
- [ ] Agree on technical approach with reviewer (if the changes are nuanced)
- [ ] Ensure that there is a testing strategy (and documented non-automated tests)
- [ ] Ensure there is a documented monitoring strategy (if necessary)
- [ ] Consider systems implications

### CODE REVIEW

File: shared/data/etags.json [hunk 1]
```diff
@@ -1 +1 @@
-{"config-etag":"W/\"09fa14b9e65122d55e9f024922140627\""}
\ No newline at end of file
+{"config-etag":"W/\"32052367-40070-2022-05-27T19:47:39.127Z\""}
```
Comment (SlayterDev) [inline shared/data/etags.json:1]: This doesn't look right

Comment (jonathanKingston) [reply shared/data/etags.json:1]: You're right it doesn't!? this is pulled from the remote resource headers... maybe there's a bug in that code as this is a timestamp along with other things.

Comment (jonathanKingston) [reply shared/data/etags.json:1]: Oh wait it could be an artefact of me serving it locally and bundling from localhost!

**Review by SlayterDev — APPROVED** [review]
Send it 🚀

**Pull request merged**